controller/volume_attachment_controller.go

```diff
@@ -565,6 +572,8 @@ func (vac *VolumeAttachmentController) handleVolumeAttachment(va *longhorn.Volum
 		return
 	}
 
+	log.Infof("Volume %v is selected to attach to node %v", vol.Name, attachmentTicket.NodeID)
```

Comment:
NIT: Print out the ticket info (type) as well

Reply:
Deference and log the volumeattachement spec.AttachmentTickets and status.AttachmentTicketStatuses.
Thank you.

Reply:
The log will be like
```
time="2023-10-13T06:34:35Z" level=info msg="Volume pvc-1418f411-ccad-4df9-a66a-050cb80d4645 is selected to attach to node rancher60-worker3" VolumeAttachmentTicketStatus="(map[string]*v1beta2.AttachmentTicketStatus) (len=1) {\n (string) (len=68) \"csi-cf8cd2fc24cc334ee5115cf1b96e363c54b43efe8f8bc240c075d5e7014a445d\": (*v1beta2.AttachmentTicketStatus)(0xc00404bb40)({\n  ID: (string) (len=68) \"csi-cf8cd2fc24cc334ee5115cf1b96e363c54b43efe8f8bc240c075d5e7014a445d\",\n  Satisfied: (bool) false,\n  Conditions: ([]v1beta2.Condition) (len=1 cap=1) {\n   (v1beta2.Condition) {\n    Type: (string) (len=9) \"Satisfied\",\n    Status: (v1beta2.ConditionStatus) (len=5) \"False\",\n    LastProbeTime: (string) \"\",\n    LastTransitionTime: (string) (len=20) \"2023-10-13T06:34:07Z\",\n    Reason: (string) \"\",\n    Message: (string) (len=40) \"Waiting for volume share to be available\"\n   }\n  },\n  Generation: (int64) 0\n })\n}\n" VolumeAttachmentTickets="(map[string]*v1beta2.AttachmentTicket) (len=2) {\n (string) (len=68) \"csi-cf8cd2fc24cc334ee5115cf1b96e363c54b43efe8f8bc240c075d5e7014a445d\": (*v1beta2.AttachmentTicket)(0xc00404bac0)({\n  ID: (string) (len=68) \"csi-cf8cd2fc24cc334ee5115cf1b96e363c54b43efe8f8bc240c075d5e7014a445d\",\n  Type: (v1beta2.AttacherType) (len=12) \"csi-attacher\",\n  NodeID: (string) (len=17) \"rancher60-worker3\",\n  Parameters: (map[string]string) (len=2) {\n   (string) (len=14) \"lastAttachedBy\": (string) \"\",\n   (string) (len=15) \"disableFrontend\": (string) (len=5) \"false\"\n  },\n  Generation: (int64) 0\n }),\n (string) (len=65) \"share-manager-controller-pvc-1418f411-ccad-4df9-a66a-050cb80d4645\": (*v1beta2.AttachmentTicket)(0xc00404bb00)({\n  ID: (string) (len=65) \"share-manager-controller-pvc-1418f411-ccad-4df9-a66a-050cb80d4645\",\n  Type: (v1beta2.AttacherType) (len=24) \"share-manager-controller\",\n  NodeID: (string) (len=17) \"rancher60-worker3\",\n  Parameters: (map[string]string) (len=1) {\n   (string) (len=15) \"disableFrontend\": (string) (len=5) \"false\"\n  },\n  Generation: (int64) 0\n })\n}\n" controller=longhorn-volume-attachment longhornVolumeAttachment=pvc-1418f411-ccad-4df9-a66a-050cb80d4645 node=rancher60-worker3
```

A little lengthy, but it can help us understand the tickets and status.

Reply:
Simplified the log. Thank you.